packages/sdk/package.json

```diff
@@ -26,15 +26,15 @@
     "integration-test:browser-case": "karma start --single-run --glob=./dist-test/index.integration.browser.js",
     "integration-test:browser": "node test/integration/scripts/DownloadSimpleAuth.js && concurrently --kill-others --success first \"node test/integration/scripts/StartSimpleAuth.js\" \"npm run integration-test:browser-case\"",
     "integration-test:node": "mocha -r test/mocha.env.ts -r ts-node/register \"./test/integration/node/*.spec.ts\" --timeout 100000 --exit --reporter ./tools/mocha-multi-reporter.js",
-    "test:integration": "npm run integration-test:node && npm run integration-test:browser",
+    "test:e2e": "npm run integration-test:node && npm run integration-test:browser",
```

Comment:
if the test:integration rename to test:2e2, the integraiont-test:browser should be rename to e2e-test:browser too?

Reply:
Yes, I'll rename "integration" wording.